aemedge/blocks/accordion/accordion.css

```diff
@@ -103,5 +104,6 @@
     margin: 0 auto 4rem;
     max-width: 780px;
     width: 66%;
+    padding: 0 1rem 0 5rem;
   }
```

Comment:
the `width: 66%` on line 105 works well for standalone accordions, but when in tabs, it is too narrow.
Try adding a new style:
```
.tabs .accordion-wrapper {
width: unset;
}
```
![image](https://github.com/user-attachments/assets/99cbf986-5fd6-4b24-838b-ba94bc6c1e49)